src/libstore/content-address.cc

```diff
@@ -155,7 +174,7 @@ std::string renderContentAddress(std::optional<ContentAddress> ca)
 
 std::string ContentAddress::printMethodAlgo() const
 {
-    return method.renderPrefix()
+    return std::string { method.renderPrefix() }
```

Comment:
need this boilerplate because new return type, we cannot add together `std::string_view`s, only had a `std::string_view` to a `std::string`.